content/04.results.md

```diff
@@ -2,42 +2,54 @@
-In other cases (apparent with CtD), one label function appears to improve performance; however, adding more label functions does not improve performance (AUROC) or decreases it (AUPR).
-In certain cases, the source of the label functions appears to be important: the performance of CbG edges decrease when using label functions from the DaG and CtD categories.
-
-Our initial hypothesis was based on the idea that certain edge types capture similar physical relationships and that these cases would be particularly amenable for label function transfer.
-For example, CbG and GiG both describe physical interactions.
-We observed that performance increased as assessed by both AUROC and AUPR when using label functions from the GiG edge type to predict CbG edges.
-A similar trend was observed when predicting the GiG edge; however, the performance differences were small for this edge type making the importance difficult to assess.  
-The last column shows increasing performance (AUROC and AUPR) for both DaG and CtD when sampling from all label functions.
-CbG and GiG also had increased performance when one random label function was sampled, but performance decreased drastically as more label functions were added.
-It is possible that a small number of irrelevant label functions are able to overwhelm the distant supervision label functions in these cases (see Figure {@fig:auroc_random_label_function_performance} and Supplemental Figure {@fig:aupr_random_label_function_performance}).
-
-### Random Label Function Generative Model Analysis
+We reported the results in AUROC and AUPR (Figures {@fig:auroc_gen_model_test_set} and {@fig:aupr_gen_model_test_set}; Supplemental Figures {@fig:auroc_gen_model_tune_set} and {@fig:auroc_gen_model_tune_set}).  
+Performance increases as the number of edge-specific label functions is added to the baseline model (Figures {@fig:auroc_gen_model_test_set} and {@fig:aupr_gen_model_test_set}). 
+The Compound treats Disease (CtD) edge type is a quintessential example of this trend.
+The baseline model starts off with an AUROC score of 52% and an AUPRC of 29%, then performance increases to 76% and 48% respectively as more CtD related label functions are included. 
+Gene interacts Gene (GiG), Disease associates Gene (DaG) and Compound binds Gene (CbG) edge types share a similar trend.
+Baseline performance starts off with an AUROC of 57% for DaG, 78% for CbG and 66% for GiG, then increases to 62% for DaG, 81% for CbG and 75% for GiG.
+Similarly with AUPR, the baseline starts off at 43% for DaG, 29% for CbG and 30% for GiG, then increases to 44%, 30% and 35% respectively.
+
+We based our initial hypothesis on the idea that certain edge types would share related linguistic features and terminology which would be amenable to label function transfer.
+For example, sentences for edges CbG and GiG typically describe physical interactions whether it be between a compound and a gene or a pair of two protein produced from genes.
```

Comment:
Train wifi is dying on me but I couldn't recall if this feature was retained or went away. Since I closed the figure and now can't re-load it, is this still the case?

Reply:
Ok - it's working again - is it weird that DAG gets better from every edge type?